setup.py

```diff
@@ -49,7 +49,7 @@
     packages=find_packages(include=["unyt"]),
     test_suite="tests",
     tests_require=test_requirements,
-    python_requires=">=3.6.1",
+    python_requires=">=3.8.10",
```

Comment:
```suggestion
    python_requires=">=3.8",
```

Do you really want to **force** people to be on CPython `3.8.10+`? Unless you actually are relying on a patch in CPython itself this feels aggressively restrictive as someone who had an older Python 3.8 that would be able to use the software is now barred from installing this version. Why lock people out unnecessarily?